packages/web-components/src/components/tabs-extended/tabs-extended.ts

```diff
@@ -32,23 +32,25 @@ class DDSTabsExtended extends StableSelectorMixin(LitElement) {
    * Child tab components.
    */
   @internalProperty()
-  private _tabItems: Node[] = [];
+  private _tabItems: DDSTab[] = [];
 
   /**
    * Defines the active tab index.
    */
   @internalProperty()
-  private _activeTab: Number = 0;
+  private _activeTab: number = 0;
+
+  @internalProperty()
+  private _isLTR: boolean = false;
```

Comment:
should we be using `_isRTL` instead? I believe most pages will be `LTR` so making the default value to false doesn't make sense to me

Reply:
Also testing out the WC RTL deploy preview, it seems to be focusing on the opposite element? When I'm focused on the `First Tab` of `tabs-extended` and I pressed the left arrow, I expect the focus to be on the `Second Tab` but instead it focuses on the `Fourth Tab`

Reply:
🤦 I started it as isRTL, then switched to isLTR to make the code more easily readable. Thoughts on making this property default to `true` instead?

Also, the property sets in `updated`, but I wasn't sure if maybe it would be better in `connectedCallback`? We probably don't anticipate much switching between RTL/LTR after page load, do we?

Reply:
ah ok, yeah let's make the `isLTR` default to `true` instead and I'm fine with it in the `updated()` since yeah I don't think there will be much switching between RTL and LTR.

Also, is the RTL navigation I mentioned in the comment above the expected behavior?

Reply:
@annawen1 can you double check the WC RTL deploy preview? Maybe it hadn't finished updating with new code, but it's working for me now.

Reply:
@andy-blum oh yep, opened it in incognito and it works as expected now